eox_core/api/v1/api_schema.py

```diff
@@ -0,0 +1,59 @@
+"""
+Swagger view generator
+"""
+from django.conf import settings
+from django.conf.urls import url
+from drf_yasg.generators import OpenAPISchemaGenerator
+from drf_yasg.openapi import SwaggerDict
+from drf_yasg.views import get_schema_view
+from edx_api_doc_tools import get_docs_cache_timeout, make_api_info
+from rest_framework import permissions
+
+from eox_core.api.v1 import views
+
+
+class APISchemaGenerator(OpenAPISchemaGenerator):
+    """
+    Schema generator for eox-core.
+
+    Define an exclusive base path to perform requests for each endpoint and a
+    specific security definition using oauth without overwritting project wide
+    settings.
+    """
+
+    def get_schema(self, request=None, public=False):
+        schema = super().get_schema(request, public)
+        schema.base_path = '/eox-core/api/v1/'
```

Comment:
Since '/eox-core/api/v1/' is being used more than once maybe you should consider declaring a constant at the beginning of the file.

Reply:
This one is actually an oversight, the second time I use that path it's not doing anything so I'm going to delete it.

Reply:
ok